### DISCUSSION TITLE
The failure screen of a failed replay can be a little confusing

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#22293] [Q&A] [answered]
The replay failure screen is a bit confusing since it's currently identical to the gameplay one.
Some examples:
- The "retry" button may suggest that you are going to play the map after seeing the replay, when in fact you are just rewatching it
- The retry counter I guess doesn't make much sense since it simply counts how many times you rewatched that replay in that session

I hope you can make it different in the future. Some random ideas:
- Change the style/colour of the buttons (or just show "rewatch" instead of "retry")
- Change title and/or subtitle (e.g. "`{username} failed`")
- Hide the button to save/watch the replay (since it's already saved and clicking it restarts the replay)
- Show the number of user retries for that replay (not sure if it's feasible to retrieve it as a statistic)

![image](https://user-images.githubusercontent.com/43073074/213453748-21d8f952-3e44-4131-a9a4-02219fc2d812.png)

**Comment by bdach** [comment]
Agree that this screen is pretty bad. All points raised are valid to me.

**Comment by peppy** [comment, marked as answer]
Tracking at https://github.com/ppy/osu/issues/22298
